phpBB/config/default/container/services.yml

```diff
@@ -25,6 +25,7 @@ imports:
     - { resource: services_profilefield.yml }
     - { resource: services_report.yml }
     - { resource: services_routing.yml }
+    - { resource: services_storage.yml }
```

Comment:
please removes the services, we will commit this part later as it is not really designed for now